src/Symfony/Component/Form/Extension/Core/DataMapper/PropertyPathMapper.php

```diff
@@ -18,10 +18,14 @@
 use Symfony\Component\PropertyAccess\PropertyAccess;
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 
+trigger_deprecation('symfony/form', '5.2', 'The "%s" class is deprecated.', PropertyPathMapper::class);
```

Comment:
we should tell about the alternative in the message and in the annotation:
use `DataMatter` instead

Reply:
done, thanks!